tests/test_reverse_mode.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+
+from GrADim.reverse_mode import ReverseMode
+from GrADim.GrADim import Gradim
+
+
+# Run using """python -m pytest tests/test_reverse_mode.py"""
+class TestReverseMode:
+    x = ReverseMode(3)
+    y = ReverseMode(2)
+    float_equality_threshold = 1e-8
+
+    def test_addition_c():
+    g = x + 2
+    assert (float(g.value) == 5.0) & (float(x.derivative) == 1.0)
```

Comment:
Shouldn't there be an indent here (and for all the other functions)? 
Also, are you sure that using & as a boolean operator works ? I would have just used "and"
As all the functions are within the class, you should have self as argument everywhere and then use self.x and self.y every time you want to call an element of the class